packages/amplication-data-service-generator/src/server/swagger/swagger.template.ts

```diff
@@ -13,7 +13,7 @@ export const swaggerDocumentOptions = new DocumentBuilder()
   .AUTH_FUNCTION()
   .build();
 
-export const swaggerSetupOptions = {
+export const customOptions: SwaggerCustomOptions = {
```

Comment:
Please don't rename the const, only add the type.

Reply:
any updates @Smartmind12 ?